packages/mwp-app-render/src/util/socialHelper.js

```diff
@@ -0,0 +1,40 @@
+export const getSocialLinks = locale => {
```

Comment:
Not sure if this is the right spot for this. Could also use some docs.

Reply:
looks like this exists in meetup-web-components [here](https://github.com/meetup/meetup-web-components/blob/master/src/utils/getSocialLinks.js)

Reply:
Ah! It's also in `mup-web` here - https://github.com/meetup/mup-web/blob/master/src/util/socialHelper.js

I think I'll get a ticket in to deal with this.